snippets/nuxt-script.json

```
@@ -0,0 +1,27 @@
+{
+  "Nuxt Async": {
+    "prefix": "nasyncdata",
+    "body": ["async asyncData ({ ${1:params} }) {", "\tconst { data } = await axios.get(`${2:endpoint}`)", "\treturn { ${3:key}:${4:value} }", "},"],
```

```suggestion
    "body": ["async asyncData ({ ${1:params} }) {", "\tconst { data } = await fetch(`${2:endpoint}`).then(res => res.json())", "\treturn { ${3:key}:${4:value} }", "},"],
```

Using `fetch` will allow to remove `axios` dependency :)